extending-valtimo/access-control/creating-a-resource.md

```diff
@@ -0,0 +1,101 @@
+# PBAC resource
+
+PBAC allows admins to configure which resources a user can access. For example, it is configurable if a person with the
+role 'ROLE_USER' has a 'view' permission on the 'CrimeScene' resource. Unfortunately, not all resources are supported by PBAC.
+But PBAC does allow new resources to be added.
+
+## Supported resources
```

Comment:
I think this section needs to be changed completely. It should also reference the reference section on supposed resources types.

Reply:
Updated it. Should add a link to the references section when that's added, but that can also be done as a separate PR.